docker/ludwig-dev/Dockerfile

```diff
@@ -10,6 +10,7 @@
 #   unit testing
 #
 
+FROM python:3.8.12-slim
```

Comment:
Hmm, I forgot about this image. Looks like we need to update the parent on line 15 to match `ludwig-gpu` (really the whole image should match except line 33, which should indeed be using test).